docker-compose-deps-1.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+services:
+  mongo-db:
+    container_name: mongo-db
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- 2c312e5b1efa92900c802910d609d4fa -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Improper Privilege Management
_High CWE-269_
The Docker container requests the `NET_RAW` capability, either explicitly or by default, granting access to the host's network interfaces.



### How to fix
Potential mitigations include the following:
  1. Very carefully manage the setting, management, and handling of privileges. Explicitly manage trust zones in the software.
  2. Follow the principle of least privilege when assigning access rights to entities in a software system.
  3. Consider following the principle of separation of privilege. Require multiple conditions to be met before permitting access to a system resource.